python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/conftest.py

```diff
@@ -381,6 +382,16 @@
 }
 
 
+@pytest.fixture(name="api_key")
+def api_key_fixture() -> str:
```

Comment:
nit; I feel like this makes more sense to be a constant than a fixture, personally. I feel like when I see a mountain of fixtures, I get overwhelmed by a test. I know that this doesn't originate with this PR, but I personally prefer TEST_API_KEY, TEST_API_SECRET, etc

Reply:
That's fair - done in [ad9fd4e](https://github.com/dagster-io/dagster/pull/25797/commits/ad9fd4ed50bb99bae35ff30b5d8775f5277a7f77)